pkg/antctl/raw/multicluster/delete/member_token.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package delete
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"antrea.io/antrea/pkg/antctl/raw/multicluster/common"
+)
+
+type deleteTokenOptions struct {
+	namespace string
+	k8sClient client.Client
+}
+
+var deleteTokenOpts *deleteTokenOptions
+
+var deleteTokenExamples = strings.Trim(`
+# Delete a member token in the antrea-multicluster Namespace
+  $ antctl mc delete membertoken cluster-east-token -n antrea-multicluster
+`, "\n")
+
+func (o *deleteTokenOptions) validateAndComplete(cmd *cobra.Command) error {
+	if o.namespace == "" {
+		return fmt.Errorf("Namespace is required")
+	}
+
+	var err error
+	if o.k8sClient == nil {
+		o.k8sClient, err = common.NewClient(cmd)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func NewMemberTokenCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "membertoken",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Delete a member token in a leader cluster",
+		Long:    "Delete a member token in a leader cluster, corresponding ServiceAccount and a RoleBinding will also be deleted",
```

Comment:
```suggestion
		Long:    "Delete a member token in a leader cluster, corresponding ServiceAccount and RoleBinding will also be deleted if they exist.",
```
Or you may refer to jianjun's description:
```
Delete a member token in a leader cluster. It deletes the Secret, ServiceAccount and RoleBinding for the token if they exist.
```